### PR TITLE
Cm 1044 personal value representation

### PR DESCRIPTION
**Pull request by danmash**
There are two endpoints with topMatchValue and I don't know which one is used on that page so I've changed both.

/alignment/<alignment_scores_uuid>
/alignment/<alignment_scores_uuid>/summary

Thus we need to test the usage of these two endpoints. 

The problem is that we are storing the personal value `key` in the database in `AlignmentScore` object, (which is not good btw). This key should be transformed into some representation. I did this in `PersonalVaue` since it's the single place for all `PersonalValue` stuff. 

@cyplas I've tried to change the response example of these two endpoints in stoplight but it affected a lot of fields, I don't know why. could you have a look?
Screenshot 2022-05-15 at 22 42 01

### CODE REVIEW

File: app/static/Climate-Mind_bundled.yml [hunk 1]
```diff
@@ -421,7 +414,7 @@ paths:
                   value:
                     overallSimilarityScore: 70
                     topMatchPercent: 90
-                    topMatchValue: benevolence
+                    topMatchValue: Benevolence
```
Comment (danmash) [inline app/static/Climate-Mind_bundled.yml:417]: This is my change, I don't know why there are so much stuff committed with it

File: app/static/Climate-Mind_bundled.yml [hunk 2]
```diff
@@ -1360,23 +1237,20 @@ paths:
                 Successful GET Alignment Summary Response Body:
                   value:
                     userAName: Stevie
-                    topMatchValue: benevolence
+                    topMatchValue: Self Direction
```
Comment (danmash) [inline app/static/Climate-Mind_bundled.yml:1240]: And this is also my change

**Comment by cyplas** [comment]
> @cyplas I've tried to change the response example of these two endpoints in stoplight but it affected a lot of fields, I don't know why. could you have a look?

@danmash I'm not sure what you mean. The only difference I see here between the develop and CM-1044 branches is in the topMatchValue, in line with the diff you show below. Maybe your browser is doing something funny?

**Comment by danmash** [comment]
@cyplas please have a look at the diff I see, it's enormous. 
[Cm 1044 personal value representation by danmash · Pull Request #325 · ClimateMind_climatemind-backend.pdf](https://github.com/ClimateMind/climatemind-backend/files/8703448/Cm.1044.personal.value.representation.by.danmash.Pull.Request.325.ClimateMind_climatemind-backend.pdf)

**Comment by cyplas** [comment]
> @cyplas please have a look at the diff I see, it's enormous.

@danmash Indeed, the diff is big, but you mentioned changed fields, and on quick perusal, I don't see that (neither in the diff nor on the webpage). It looks to me like Stoplight has changed how it's formatting the yml file (whitespace, quotes, etc), and it has done so also in at least one of my stoplight commits. In short, I don't know what's up with this, but I suggest if it doesn't keep happening "randomly", we don't worry about it. Unless you've noticed it changing things that aren't just harmless reformatting ...

**Comment by danmash** [comment]
> > @cyplas please have a look at the diff I see, it's enormous.
> 
> 
> 
> @danmash Indeed, the diff is big, but you mentioned changed fields, and on quick perusal, I don't see that (neither in the diff nor on the webpage). It looks to me like Stoplight has changed how it's formatting the yml file (whitespace, quotes, etc), and it has done so also in at least one of my stoplight commits. In short, I don't know what's up with this, but I suggest if it doesn't keep happening "randomly", we don't worry about it. Unless you've noticed it changing things that aren't just harmless reformatting ... 

If so, could you please approve this pr?

**Comment by danmash** [comment]
@rodriguesk I think this is ready to merge

**Review by rodriguesk — APPROVED** [review]

File: app/static/openapi.yaml [hunk 3]
```diff
@@ -1,15 +1,13 @@
...
+  description: The API for sending and recieving information to and from the ClimateMind ontology.
+  version: '0.1'
 servers:
-  - url: http://127.0.0.1:5000
-  - url: http://localhost:5000
+  - url: 'http://127.0.0.1:5000'
+  - url: 'http://localhost:5000'
 paths:
-  /scores:
+  /alignment:
```
Comment (rodriguesk) [inline app/static/openapi.yaml:10]: This should remain /scores. It should not be /alignment. Right @danmash ?

Comment (danmash) [reply app/static/openapi.yaml:10]: you are right

**Comment by rodriguesk** [comment]
@danmash it still looks like /scores is changed to /alignment ?

**Review by rodriguesk — CHANGES_REQUESTED** [review]
/alignment needs to be changed back to say /scores in app/static/openapi.yaml as we agreed in our github comments conversation.

**Comment by danmash** [comment]
@rodriguesk I can't do this in stoplight

Screenshot 2022-05-18 at 23 46 31

**Comment by rodriguesk** [comment]
@danmash endpoint documentation changes must be separate branches than coding changes. Would it help to create a separate branch from this documentation change?

**Comment by danmash** [comment]
I've just updated the file manually, please merge the branch

**Review by rodriguesk — APPROVED** [review]

**Pull request merged**